app/lib/service/csp/backend.dart

```diff
@@ -19,30 +21,37 @@ CspBackend get cspBackend => ss.lookup(#_cspBackend) as CspBackend;
 
 /// Represents the backend for the CSP handling and related utilities.
 class CspBackend {
```

Comment:
This ended up not working anyways... I'm okay with removing the feature too.

Reply:
I'll remove it in a follow-up.